File: src/test/java/FileReaderTest.java
import java.io.File;
import java.io.FileWriter;
import java.io.IOException;

import static org.junit.jupiter.api.Assertions.*;

class FileReaderTest {

    @org.junit.jupiter.api.Test
    void readFileContent() {
        try {
            FileWriter fileWriter = new FileWriter("src/main/resources/temp.txt");
            fileWriter.write("hello world!");
            fileWriter.close();
        } catch (IOException e) {
            e.printStackTrace();
        }

        File tempFile = new File("src/main/resources/temp.txt");
        String content = FileReader.readFileContent(tempFile);

        assertEquals(content, "hello world!");

        tempFile.delete();
    }

    @org.junit.jupiter.api.Test
    void readNotExistingFileContent() {
        assertDoesNotThrow(() ->{
            FileReader.readFileContent(new File("src/main/resources/notExistingFile.xyz"));
        });
    }
}

File: Phase5/SampleLibrary/SampleLibrary.Test/FileReaderTest.cs
using System.IO;
using Xunit;

namespace SampleLibrary.Test
{
    public class FileReaderTest
    {
        [Fact]
        private void ReadFileContentTest()
        {
            const string tempFileName = "temp.txt";
            const string tempFileContent = "hello world!";

            File.WriteAllTextAsync(tempFileName, tempFileContent);

            var content = FileReader.ReadFileContent(tempFileName);

            Assert.Equal(tempFileContent, content);

            File.Delete(tempFileName);
        }

        [Fact]
        private void ReadNotExistingFileContentTest()
        {
            const string notExistingFilePath = "src/main/resources/notExistingFile.xyz";
            Assert.Equal("", FileReader.ReadFileContent(notExistingFilePath));
        }
    }
}